### PR TITLE
added thoth-license-solver

### PR DESCRIPTION
**Pull request by meile18**
## This introduces a breaking change

- [x] No

## This should yield a new module release

- [x] Yes

## This Pull Request implements

added `license-solver`

Adds:
- new table `python_package_license`
- new columns in `python_package_version` for `python_package_license`

## Description
into python_package_version will be saved normalized license namings

### CODE REVIEW

**Review by KPostOffice — COMMENTED** [review]
This /lgtm , another set of eyes would be nice though

**Review by harshad16 — APPROVED** [review]
/lgtm
/approve
thanks :100:

**Comment by goern** [comment]
/approve
:100: nice to see this part move forward!!

File: thoth/storages/data/alembic/versions/2a4632197512_added_license_solver.py
Comment (mayaCostantini) [inline thoth/storages/data/alembic/versions/2a4632197512_added_license_solver.py:32]: I think the `constraint_name` parameter is mandatory here, see the [alembic documentation](https://alembic.sqlalchemy.org/en/latest/ops.html#alembic.operations.BatchOperations.create_foreign_key).

File: thoth/storages/graph/models.py
Comment (mayaCostantini) [inline thoth/storages/graph/models.py:102]:
```suggestion
    """Representation of a License."""
```

**Review by mayaCostantini — COMMENTED** [review]
looks really good :100: Just a few comments
Have you tested those changes locally?

**Comment by mayaCostantini** [comment]
ping @meile18 :)

**Review by mayaCostantini — APPROVED** [review]
Great work! :1st_place_medal:  lgtm, please squash your commits before merging

**Pull request merged**
